### PR TITLE
plan: refine explain result format

### PR DESCRIPTION
**Pull request by coocood**
## What have you changed? (mandatory)

Delete redundant IndexLookUp description and move the `cost` column next to `id` to make it more noticeable.

## What are the type of the changes (mandatory)?

- Improvement (non-breaking change which is an improvement to an existing feature)

## How has this PR been tested (mandatory)?

unit test

### CODE REVIEW

**Comment by zz-jason** [comment]
LGTM

File: plan/explain.go [hunk 1]
```diff
@@ -116,7 +116,7 @@ func (p *PhysicalIndexReader) ExplainInfo() string {
 
 // ExplainInfo implements PhysicalPlan interface.
 func (p *PhysicalIndexLookUpReader) ExplainInfo() string {
-	return fmt.Sprintf("index:%s, table:%s", p.indexPlan.ExplainID(), p.tablePlan.ExplainID())
+	return ""
```
Comment (zhexuany) [inline plan/explain.go]: Maybe better to  add comment to explain why only return `""` to its caller?

**Comment by zhexuany** [comment]
Well Done. Except the comment of function. Other Part looks good to me. If you resolve this, just ping me so that I can approve this PR. ;)

**Pull request merged**